learn/how-tos/building-pwa-app.md

```diff
@@ -0,0 +1,123 @@
+---
+title: "Enabling PWA - Beta"
+id: "building-pwa-app"
+sidebar_label: "Enabling PWA"
+---
+---
+
+## What is PWA (Progressive Web Application)
+
+PWA (Progressive Web Application) are web applications that provide a native mobile app experience. These applications combine great functionalities and enhanced user experience based on the browser’s capabilities, and works on any platform, including desktop and mobile devices. You can develop PWA using web technologies such as JavaScript, HTML, and CSS.
+
+### Advantages of Using PWA
```

Comment:
We should add section above this with title:

When should i use PWA vs WaveMaker Mobile application